test/manifold/stream_test.clj

```diff
@@ -478,6 +479,22 @@
       (is (= (range 1 11)
              (s/stream->seq sliding-s)))))  )
 
+
+(deftest test-blocking-queue-close
+  (testing "BlockingQueueSource reports when closed"
+    (let [k (s/->source (LinkedBlockingQueue. [0 1 2 3]))]
+      (is (= 0 @(s/take! k)))
+      (is (= 1 @(s/take! k)))
+      (is (= 2 @(s/take! k)))
+
+      (s/close! k)
+
+      (is (= ::closed @(s/try-take! k ::closed 100 ::timeout)))
+
+      )
+    )
+  )
```

Comment:
Style note: these should go at the end of the final line. Check out https://guide.clojure.style/#gather-trailing-parens